<?xml version="1.0"?>
<launch>
	<node pkg="redone" name="search_excute_node" type="search_excute_node" output="screen">
		<param name="db_ip" type="str" value="192.168.0.199" />
		<param name="db_port" type="int" value="11000" />
	</node>
</launch>
